### PR TITLE
[MWPW-160011] - GNAV light mode color changes

### PR DESCRIPTION
**Pull request by Deva309**
The Lightmode colors between the Milo GNav and Adobe Home GNav are different.
So Added following changes to Milo GNav and provide a more standard experience around the ecosystem.

Resolves: [MWPW-160011](https://jira.corp.adobe.com/browse/MWPW-160011)
Bug Fix: [MWPW-161889](https://jira.corp.adobe.com/browse/MWPW-161889)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://mwpw-160011--milo--deva309.hlx.page/?martech=off

QA:

- https://adobecom.github.io/nav-consumer/navigation.html?env=stage&navbranch=mwpw-160011&authoringpath=/federal/test-website (Standalone GNAV)
- https://main--cc--adobecom.hlx.page/drafts/devashish/vh?milolibs=mwpw-160011--milo--deva309 (CC)

### CODE REVIEW

**Review by sharmrj — APPROVED** [review]

**Review by bandana147 — APPROVED** [review]

**Review by nishantka — APPROVED** [review]

**Comment by spadmasa** [comment]
Validations has been done on the PR detailed results are added in https://jira.corp.adobe.com/browse/MWPW-160011 and in subtask
{35B2757C-5DF6-4293-A664-60E3265B984D}
{7EDE663A-5008-4023-A1FD-370CD1BAF61F}
{2CB31E28-EF74-4F4B-9C94-733A3EB13960}
{8CBFEAA0-7833-42D9-9DFE-FD1F876F9F01}
{4064846D-407E-4290-989C-3CF74D0CF7F2}
{71679EFE-94E5-46C8-AB02-7711876F7240}
{4CF26EF5-8238-4599-A037-15D0B302FD5F}

**Comment by salonijain3** [comment]
This is a high priority PR as it is a dependency for the Standalone Gnav go live on Adobe Home.

**Comment by spadmasa** [comment]
PR needs to go live by 28th October , as there is review going on the changes removed verified label. Label will be added once we have the confirmation

File: libs/blocks/global-navigation/base.css [hunk 1]
```diff
@@ -23,11 +24,11 @@
...
   /* Links */
-  --feds-background-link--hover: #f5f5f5;
-  --feds-borderColor-link: #f3f3f3;
-  --feds-color-link: #2c2c2c;
-  --feds-color-link--hover: #1473e6;
-  --feds-color-navLink-description: #656565;
+  --feds-background-link--hover: #e9e9e9;
+  --feds-borderColor-link: #e9e9e9;
+  --feds-color-link: #292929;
+  --feds-color-link--hover: #274DEA;
```
[the remark this answers is not in the document]
Comment (Deva309) [reply libs/blocks/global-navigation/base.css:30]: Updated to Uppercase everywhere.

File: libs/blocks/global-navigation/dark-nav.css [hunk 2]
```diff
@@ -11,6 +11,7 @@
   --feds-borderColor: #303030;
   --feds-borderColor-menu: #4B4B4B;
   --feds-borderColor-navLink: #DBDBDB;
+  --feds-background-promo: #1D1D1D;
```
[the remark this answers is not in the document]
Comment (Deva309) [reply libs/blocks/global-navigation/dark-nav.css:14]: Updated to Uppercase everywhere.

File: libs/blocks/global-navigation/base.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (salonijain3) [reply libs/blocks/global-navigation/base.css:30]: @Deva309 can you update to lower case as suggested?

[the remark this answers is not in the document]
Comment (Deva309) [reply libs/blocks/global-navigation/base.css]: @salonijain3 updated to lowercase for all css variables.

File: libs/blocks/global-navigation/dark-nav.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (Deva309) [reply libs/blocks/global-navigation/dark-nav.css]: @salonijain3 updated to lowercase for all css variables.

**Pull request merged**